domains/guess-who.is-cool.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Personal website",
+
+    "domain": "is-cool.dev",
+    "subdomain": "guess-who",
+
+    "owner": {
+        "email": "rayrsn@proton.me"
+    },
+
+    "record": {
+        "NS": ["ns1.epizy.com", "ns2.epizy.com"]
```

Comment:
As per our [notice](https://github.com/open-domains/register#notice), we no longer support NS records.